recipes/aws-c-common/all/conanfile.py

```diff
@@ -67,10 +68,18 @@ def package(self):
         tools.rmdir(os.path.join(self.package_folder, "lib", "aws-c-common"))
 
     def package_info(self):
-        self.cpp_info.libs = ["aws-c-common"]
+        self.cpp_info.filenames["cmake_find_package"] = "aws-c-common"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "aws-c-common"
+        self.cpp_info.names["cmake_find_package"] = "AWS"
+        self.cpp_info.names["cmake_find_package_multi"] = "AWS"
+        self.cpp_info.components["aws-c-common-lib"].names["cmake_find_package"] = "aws-c-common"
+        self.cpp_info.components["aws-c-common-lib"].names["cmake_find_package_multi"] = "aws-c-common"
+        self.cpp_info.components["aws-c-common-lib"].libs = ["aws-c-common"]
         if self.settings.os == "Linux":
-            self.cpp_info.system_libs = ["m", "pthread", "rt"]
+            self.cpp_info.components["aws-c-common-lib"].system_libs = ["m", "pthread", "rt"]
+        elif self.settings.os == "Windows":
+            self.cpp_info.components["aws-c-common-lib"].system_libs = ["bcrypt", "ws2_32"]
         if not self.options.shared:
             if tools.is_apple_os(self.settings.os):
-                self.cpp_info.frameworks = ["CoreFoundation"]
-        self.cpp_info.builddirs = [os.path.join("lib", "cmake")]
+                self.cpp_info.components["aws-c-common-lib"].frameworks = ["CoreFoundation"]
+        self.cpp_info.components["aws-c-common-lib"].builddirs = [os.path.join("lib", "cmake")]
```

Comment:
@madebr Shouldn't it be `build_modules` instead of `builddirs`?

Reply:
Yes, it should be `build_modudules`.
And it should be assigned a list of all cmake modules, instead of their parent directory.
Thanks for catching this.

Reply:
I looked at it again.
It should be `builddirs`, not `build_modules`.

- paths in `builddirs` get added to `CMAKE_MODULE_PATH`, which consumers then can `include`
- paths in `build_modules` get automatically included as well when doing a `find_package`, or doing `conan_basic_setup`.

I think we want the first here, so `builddirs` is correct.

Reply:
Ok thanks. So no modification, just waiting for a fix in hook H019.

Reply:
New PR fixing the situation: https://github.com/conan-io/conan-docker-tools/pull/221